text/0000-deprecate-ember-logger.md

```diff
@@ -0,0 +1,265 @@
+- 2018-01-17
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Deprecation of Ember.Logger
+
+## Summary
+
+This RFC provides for the deprecation and eventual removal of Ember.Logger.
```

Comment:
did you miss a work in "provides for the deprecation"?

Reply:
No, the wording is grammatical but awkward. I'll fix it.